backend/services/ephemeris_service.py

```
@@ -0,0 +1,81 @@
+import datetime
+from datetime import datetime
```

Ruff (0.11.9)

Redefinition of unused `datetime` from line 1

(F811)